lib/cli.ts

```diff
@@ -1,35 +1,142 @@
 #!/usr/bin/env node
 
 import Server from './server'
-import Zen from './index'
+import initZen, { Zen } from './index'
 import yargs from 'yargs'
 import * as Util from './util.js'
 import * as Profiler from './profiler'
 
+type testResult = {
+  fullName: string
+  attempts: number
+  error: string
+  time: number
+}
+
+export type CLIOptions = {
+  logging: boolean
+  rerun: number
+  lambdaCutoff: number
```

Comment:
I know it's from before but I wonder what the "cutoff" is about? Is it the maximum number of workers or maximum number of jobs per worker? Either way, I feel like it could be renamed more clearly. I even wonder if we need to have that as an option…

Reply:
It is a leftover from an old fix. It changes the cutoff time in the lambda, It was a more internal thing that I don't think is really needed anymore.